subjects/error_types/README.md

```diff
@@ -60,8 +66,8 @@ impl FErr {
 
 #[derive(Debug, Eq, PartialEq)]
 pub enum SexType {
```

Comment:
We should change `SexType` and `sex` to `GenderType` and `Gender`. I'll make a separate issue for that though, as the tests probably need to change too.